server.go

```diff
@@ -22,16 +23,24 @@ func ListenAndServe(network string, addr string, handler mux.Handler) error {
 		if err != nil {
 			return err
 		}
-		defer l.Close()
 		s := udp.NewServer(udp.WithMux(handler))
+		defer func() {
+			if err := l.Close(); err != nil {
```

Comment:
Wasn't sure how to handle this, so there are three suggestions in this file

1) add public `Error` method to tcp/udp.Server that handles errors
2) use `log.Printf`
3) ignore it

@jkralik what do you think, which way is the best?

Reply:
you can handle this error at line 32. If `s.Serve` returns nil then you use error from `l.Close`

Reply:
I guess that's alright, we can ignore the `Close` error if the `Serve` call fails.